### PR TITLE
Setup Wizard: Add "Description Only" fields to form

### PR DESCRIPTION
**Pull request by Abhinegi2**
closes: #2351

### Visible/Frontend Changes
- [x] Added a new field in the toolbar for the Description-only setting.
- [x] While dragging these fields it is opens a form to edit the multi-line labels.
- [x] Also after saving the data from popup window it saves the details.

### CODE REVIEW

File: src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { AdminEntityTextComponent } from "./admin-entity-text.component";
+import { MAT_DIALOG_DATA, MatDialogRef } from "@angular/material/dialog";
+import { Entity } from "app/core/entity/model/entity";
+import { FontAwesomeTestingModule } from "@fortawesome/angular-fontawesome/testing";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { FormBuilder } from "@angular/forms";
+import { AdminEntityService } from "app/core/admin/admin-entity.service";
+import { SimpleChange } from "@angular/core";
+
+fdescribe("AdminEntityTextComponent", () => {
```
Comment (sleidig) [inline src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.spec.ts:12]:
```suggestion
describe("AdminEntityTextComponent", () => {
```

Comment (sleidig) [reply src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.spec.ts:12]: please remember to undo these :wink:

Comment (Abhinegi2) [reply src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.spec.ts]: sure, will remember next time.

File: src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    ErrorHintComponent,
+    FormsModule,
+    ReactiveFormsModule,
+    FontAwesomeModule,
+    MatTooltipModule,
+  ],
+  templateUrl: "./admin-entity-text.component.html",
+  styleUrl: "./admin-entity-text.component.scss",
+})
+export class AdminEntityTextComponent implements OnChanges {
```
Comment (sleidig) [inline src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts:49]: Can you rename this to "AdminEditDescriptionOnlyFieldComponent" please? (aligning 1-to-1 with the field schema name)

**Review by sleidig — CHANGES_REQUESTED** [review]
looks good. :+1: 

Made some logical adjustments, as I want a different handling for this fields in our complex config model

File: src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    if (!this.fieldId) {
+      this.fieldId = uuid();
+    }
+
+    const updatedEntitySchema = Object.assign(
+      { _isCustomizedField: true },
+      {
+        id: this.fieldId,
+        editComponent: "EditDescriptionOnly",
+        label: $localize`:description section:` + `${formValues.label}`,
```
[the remark this answers is not in the document]
Comment (sleidig) [reply src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts:105]: (done now)

File: src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts (hunk 2 above)
Comment (Abhinegi2) [reply src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts:40]: sure let me make this adjustments

Comment (Abhinegi2) [reply src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-text/admin-entity-text.component.ts]: Updated the name of `AdminEntityTextComponent` to `AdminEditDescriptionOnlyFieldComponent`, ensuring consistency across folder names and selections.

**Review by sleidig — APPROVED** [review]

**Pull request merged**